src/TodoForm.jsx

```diff
@@ -0,0 +1,11 @@
+function TodoForm(){
+    return(
+        <>
```

Comment:
I forgot to put it here, but I would probably wrap it in a form. Other than that, it looks fine

Reply:
Thank you so much for such a prompt and valulable feedback. I'll try to incorporate it my codebase anyway!